ci/build-docker-image.sh

```diff
@@ -64,11 +64,14 @@ esac
 
 # Create tag based on branch name
 default_branch=$(git remote show origin | grep HEAD | awk '{print $3;}')
-if [ "$default_branch" == "$CI_BRANCH" ]
+CI_DOCKER_BRANCH=${CI_PULL_REQUEST_BRANCH:-$CI_BRANCH}
```

Comment:
I think the pull request branch shouldn't be used at all. As the pull request test should test the merged state. So that is the feature branch merged into the target branch. So CI branch is the correct branch in that case.

Reply:
Basically the CI_BRANCH should be checked out in the PR test as that if the branch the feature branch is being merged into. And if CI_BRANCH turns out to be default, then leave it empty. Pretty much what's being done in `install-package.sh` right?

Reply:
Yes